src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final int index = new Random().nextInt(Colors.values().length);
```

Comment:
it's not random really. now, you are creating index once and use that index all the time, so `getRandomColor` would return single color, not random. consider creating index in `getRandomColor` method